back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/admin_digest.rb

```diff
@@ -222,7 +222,7 @@ def successful_initiatives(time: Time.zone.today)
         .published
         .joins(initiative_status_changes: :initiative_status)
         .includes(:initiative_images)
-        .where(initiative_statuses: { code: 'threshold_reached' })
+        .with_status_code('threshold_reached')
```

Comment:
Fixes error in `admin_digest_spec.rb:40` that appeared after converting scopes to use new `with_status_code` scope:
```Ruby
EmailCampaigns::Campaigns::AdminDigest#generate_commands does not include native survey responses
     Failure/Error: command = campaign.generate_commands(recipient: admin).first
     
     ActiveRecord::StatementInvalid:
       PG::UndefinedTable: ERROR:  invalid reference to FROM-clause entry for table "initiative_statuses"
       LINE 1: ...WHERE "initiatives"."publication_status" = $1 AND "initiativ...
                                                                    ^
       HINT:  Perhaps you meant to reference the table alias "initiative_status".
 ```